frontend/src/locals/en.json

```diff
@@ -10,17 +10,17 @@
   "searchInputPlaceholder": "Search Chats ...",
   "defaultErrorMessage": "I'm still learning! Can you rephrase your question in a different way, or maybe I can help you with something else?",
   "typingIndicatorMessage": "Go ahead, I'm listening...",
-  "welcomeStatementStart": "Hi, I am ",
-  "welcomeStatementEnd": ". How can I help?",
```

Comment:
check and remove unused constants
"querius": "Querius",
  "queriusLogo": "Querius Logo",

Reply:
removed